client/webserver/site/src/js/markets.ts

```diff
@@ -690,7 +691,7 @@ export default class MarketsPage extends BasePage {
 
     if (!this.assetsAreSupported().isSupported) return // assets not supported
 
-    if (this.market.dex.tier < 1) return // acct suspended or not registered
+    if (!this.market?.dex || this.market.dex.tier < 1) return // acct suspended or not registered
```

Comment:
This also seems like an impossible condition to hit based on how `resolveOrderFormVisibility` is used.

Reply:
Not really, `createWallet` also calls this method and it could be that there is no dex connection by the time we use `resolveOrderFormVisibility`.